lib/explorer/series.ex

```diff
@@ -689,6 +689,33 @@ defmodule Explorer.Series do
   @spec bintype(series :: Series.t()) :: {:s | :u | :f, non_neg_integer()}
   def bintype(series), do: Shared.apply_impl(series, :bintype)
 
+  @doc """
+  Return a series with the category names of a categorical series.
+
+  Each category has the index equal to its position.
```

Comment:
Should we add a note that no order is guaranteed for the categories?

Reply:
I'm not sure. For what I could understand, the ordering is defined by the order the elements appear in the series. But this is not clear.

Internally there is a flag to the the ordering to lexical: https://pola-rs.github.io/polars/polars_core/datatypes/categorical/struct.CategoricalChunked.html#method.set_lexical_sorted

Reply:
I think it is defined like that but mostly as an implementation accident and not a guarantee? Providing fewer guarantees doesn’t hurt anyway :)

Reply:
Makes sense. Note added :+1: